webpack/scenes/ContentViews/Details/Filters/Rules/ContainerTag/AddEditContainerTagRuleModal.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { Modal, ModalVariant, Form, FormGroup, TextInput, ActionGroup, Button } from '@patternfly/react-core';
+import { addCVFilterRule, editCVFilterRule, getCVFilterRules } from '../../../ContentViewDetailActions';
+
+const AddEditContainerTagRuleModal = ({
+  onClose, filterId, selectedFilterRuleData = { name: '', id: undefined },
+}) => {
+  const { name, id } = selectedFilterRuleData;
+  const dispatch = useDispatch();
+  const [tagName, setTagName] = useState(name);
+  const [saving, setSaving] = useState(false);
+  const isEditing = name && id;
+
+  const onSubmit = () => {
+    setSaving(true);
+    if (isEditing) {
+      dispatch(editCVFilterRule(
+        filterId,
+        {
+          id,
+          name: tagName,
+          editMode: true,
+          working: false,
+          previous: {
```

Comment:
The editMode: true, working: false, previous values looks unnecessary..

Reply:
I'll test this. I saw this difference on the old UI.